docs/DeveloperGuide.md

```diff
@@ -212,13 +196,28 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 This section describes some noteworthy details on how certain features are implemented.
 
 The features mentioned are:
-- [Adding contacts with optional arguments](#add-contacts-with-optional-arguments)
-- [Finding contacts by keywords](#find-contacts-by-keywords)
-- [Deleting multiple contacts by keywords](#delete-by-keywords)
-- [Importing a JSON file](#import-json-file)
-- [Undoing / redoing command](#proposed-undoredo-feature)
-- [Input Suggestion](#input-suggestion)
-- {and so on...}
+1. Viewing help
```

Comment:
> This section describes some noteworthy details on how certain features are implemented.

Perhaps we may not want to describe all the implementations?

Reply:
yes I am listing them all here first just in case any wants to add, but we may delete later